temporal-sdk/src/test/java/io/temporal/activity/LocalActivityMethodOptionsTest.java

```diff
@@ -83,61 +84,22 @@ public void testActivityOptionsMerge() {
 
   @Test
   public void testLocalActivityMethodOptions() {
-    testEnv.registerActivitiesImplementations(new ActivityImpl());
-    TestActivity localActivity =
-        testEnv.newLocalActivityStub(TestActivity.class, defaultOps, perMethodOptionsMap);
+    testEnv.registerActivitiesImplementations(new LocalActivityImpl());
+    TestLocalActivity localActivity =
+        testEnv.newLocalActivityStub(TestLocalActivity.class, defaultOps, perMethodOptionsMap);
 
     // Check that options for method1 were merged.
-    Map<String, Duration> method1OpsValues = localActivity.method1();
+    Map<String, Duration> method1OpsValues = localActivity.localActivity1();
     Assert.assertEquals(
         defaultOps.getScheduleToCloseTimeout(), method1OpsValues.get("ScheduleToCloseTimeout"));
     Assert.assertEquals(
         methodOps2.getStartToCloseTimeout(), method1OpsValues.get("StartToCloseTimeout"));
 
     // Check that options for method2 were default.
-    Map<String, Duration> method2OpsValues = localActivity.method2();
+    Map<String, Duration> method2OpsValues = localActivity.localActivity2();
     Assert.assertEquals(
         defaultOps.getScheduleToCloseTimeout(), method2OpsValues.get("ScheduleToCloseTimeout"));
     Assert.assertEquals(
         defaultOps.getStartToCloseTimeout(), method2OpsValues.get("StartToCloseTimeout"));
   }
-
```

Comment:
Why delete all of this code? Do we need to change this test?

Reply:
I moved it to a new `TestLocalActivities` [class](https://github.com/imalao/sdk-java/pull/1/files#diff-627bbe4b6802b0cb50b0cc9e0cea47e1481e68da589537484cf246e2e772c2bb) so it can be used in other tests